### PR TITLE
Update add-ons documentation

### PR DESCRIPTION
**Pull request by Kendos-Kenlen**

### CODE REVIEW

**Comment by divarvel** [comment]
Can you remove the .idea directory?

**Comment by Kendos-Kenlen** [comment]
Done

**Comment by divarvel** [comment]
You need to add headers to the new files you've created (copy the existing)

**Comment by Kendos-Kenlen** [comment]
Done

File: commons/addons/redis.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+<td>78,08 €</td>
+</tr>
+<tr>
+<td class="cc-col__price "><span class="label cc-label__price label-info">XXL</span></td>
+<td>100</td>
+<td>2500 MB</td>
+<td>500</td>
+<td>190,71 €</td>
+</tr>
+</table
```
Comment (divarvel) [inline commons/addons/redis.md]: missing `>`

File: commons/addons/redis.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Redis add-on
+position: 9
+---
+
+# Redis
+
+## Redis plans
+
+<table class="table table-bordered table-striped dataTable"><caption>MySQL pricing plans</caption>
```
Comment (divarvel) [inline commons/addons/redis.md]: s/MySQL/Redis/

**Comment by divarvel** [comment]
Maybe these pages could be merged? I can't see the logic of this separation

    Getting started with add-ons
    Managing Add-ons
    Clever Cloud Add-ons

**Comment by divarvel** [comment]
Also, could you rename the PR? It's not just about cellar

File: commons/addons/clever-cloud-addons.md [hunk 3]
```diff
@@ -1,299 +1,92 @@
...
-* MongoDB
-* FS bukets
+An add-on can be shared by different applications to share data between them. It can be a database shared by two or
+three applications of your infrastructure for example, or they can be independent.
 
+The add-ons offer different plan to adapt to your needs. You can find details about them on the documentation page of
+the add-on.
 
-## MySQL plans
+## Available add-on
```
Comment (divarvel) [inline commons/addons/clever-cloud-addons.md]: add-ons

File: commons/addons/clever-cloud-addons.md [hunk 4]
```diff
@@ -1,299 +1,92 @@
...
-</tr>
-<tr>
-<td class="cc-col__price "><span class="label cc-label__price label-info">XXL</span></td>
-<td>16 GB</td>
-<td>900 GB</td>
-<td>2000</td>
-<td>2200 €</td>
-</tr>
-</table>
+Clever Cloud provides multiple add-on to work with you application :
```
Comment (divarvel) [inline commons/addons/clever-cloud-addons.md]: add-ons to use with your applications:

File: commons/addons/clever-cloud-addons.md [hunk 5]
```diff
@@ -1,299 +1,92 @@
...
 
-<br/><br/>If you need to import a very large dump, please send an email to <support@clever-cloud.com>.
+First, choose which *type* of add-on you want to create. See above to have a list of available add-ons and the
+corresponding pages for a description and information.
 
+Now, select the plan you need for you add-on. You can find details about the pricing, the capacity of the add-on, ...
+on this page or in the corresponding documentation page.
 
-## PostgreSQL plans
+The third step offer you to choose with which application you want to link you add-on. Linking an add-on with an
```
Comment (divarvel) [inline commons/addons/clever-cloud-addons.md]: offers

File: commons/addons/clever-cloud-addons.md [hunk 6]
```diff
@@ -1,299 +1,92 @@
...
+application add [environment variables](/admin-console/environment-variables.md) to the application.
```
Comment (divarvel) [inline commons/addons/clever-cloud-addons.md]: Linking an addon to an application will provide configuration to the application through environment variables

File: commons/addons/clever-cloud-addons.md [hunk 7]
```diff
@@ -1,299 +1,92 @@
...
-<th>Price /mo</th>
-</tr> 
-<tr>
-<td class="cc-col__price "><span class="label cc-label__price label-info">DEV</span></td>
-<td>SHARED</td>
-<td>256 MB</td>
-<td>10</td>
-<td>Free</td>
-</tr>
+The last step let you choose the name of the add-on (for example "My database") and the region where the add-on is
```
Comment (divarvel) [inline commons/addons/clever-cloud-addons.md]: lets

File: commons/addons/clever-cloud-addons.md [hunk 8]
```diff
@@ -1,299 +1,92 @@
...
+* the information tab
+* the Configuration tab
 
-1. <a href="https://dbms-adminer.clever-cloud.com/adminer/">our WebGUI (Adminer)</a>
-2. command line tool for PostgreSQL administration like `psql`
-3. any PostgreSQL client such as pgAdmin 3.
 
+### Information screen
 
+This screen sum-up the caracteristic of the selected add-on.
```
Comment (divarvel) [inline commons/addons/clever-cloud-addons.md]: sums up

**Comment by divarvel** [comment]
Last thing: 100Mib for FS buckets fair use

File: commons/addons/cellar.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
+---
+title: Cellar add-on
+position: 2
+---
+
+# Cellar <span class="cc-beta pull-right" title="Currently in private Beta version"></span>
+<div class="alert alert-hot-problems">
+  <h5>Note for Beta Version</h5>
+  <div>Cellar is currently in private Beta. If you want to use it, please send a mail at support@clever-cloud.com<br />
+  Cellar is free during the beta period with a fair use at 20 GB. No credits wil be charged under the fair use.</div>
```
Comment (divarvel) [inline commons/addons/cellar.md]: No credits will be charged during the beta period.

**Pull request merged**